src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -156,6 +156,10 @@ public JsonObject getMaterialType() {
     return materialTypeRepresentation;
   }
 
+  public JsonArray getCopyNumbers() {
```

Comment:
Personally, I would convert this to a collection and then map it back to JSON in the representation, however I don't think that is worthy of blocking this pull request.